assets/files/etc/selinux/config

```diff
@@ -0,0 +1,11 @@
+# This file controls the state of SELinux on the system.
+# SELINUX= can take one of these three values:
+#     enforcing - SELinux security policy is enforced.
+#     permissive - SELinux prints warnings instead of enforcing.
+#     disabled - No SELinux policy is loaded.
+SELINUX=permissive
```

Comment:
Do we have a bug reference to track fixing the policy issues that make this necessary?

Probably we should have a FIXME here to reference it?